level1.css

```
@@ -6,8 +6,20 @@ h1 {
     color: rgb(128, 94, 156);
 }
 
-button {
+button1 {
```

```suggestion
backbutton {
```
might be good to specify what the buttons do for readability